[Module] [Office365] Add `rule.level` and `rule.id` fields to alerts table of each drill down view

Desvelao opened the pull request:
### Description
This PR adds the `rule.level` and `rule.id` alert fields to the alerts table in the drills down of the Office 365's Panel.

### Changes
- Add `rule.level` and `rule.id` fields to the drills down
- Add `rule.description` field to the table that was missing

mpRegalado commented on public/components/overview/office-panel/config/drilldown-rules-config.tsx:
```diff
@@ -70,6 +70,9 @@ export const drilldownRulesConfig = {
                     { field: 'data.office365.Operation', label: 'Operation' },
+                    { field: 'rule.description', label: 'Description' },
+                    { field: 'rule.level', label: 'Level' },
+                    { field: 'rule.id', label: 'Rule ID' },
```
Rule information was deliberately omitted from this specific drilldown as it is a Rule Description Drilldown, every element in this table will share the same `Description`, `Level` and `Rule ID`.
It could be included in the table but ideally it would be included in the title of the drilldown, perhaps on a future iteration.

Desvelao replied:
I think I will add for consistency. Maybe we could discuss that in the next iterations.

Desvelao commented on public/components/overview/office-panel/config/drilldown-user-config.tsx:
```diff
@@ -75,6 +75,9 @@ export const drilldownUserConfig = {
                     { field: 'data.office365.ClientIP', label: 'Client IP' },
                     { field: 'rule.description', label: 'Description' },
                     { field: 'data.office365.Operation', label: 'Operation' },
+                    { field: 'rule.description', label: 'Description' },
```
Thank you. Fixed

Desvelao commented:
### Changes
- Moved the `rule.description` column to the rifht of the `timestamp` column
- Fixed duplicated the `rule.description` column in the drill-down related to the user

mpRegalado reviewed (approved):
CR: LGTM :heavy_check_mark: 
Testing: LGTM :heavy_check_mark:

eze9252 reviewed (approved):
LGTM!
